Do-Good-Get-Good/do-good-get-good
# Chore/update react native

This PR updates React Native and some other packages to the latest version.

## **How to test**
* Build the Android app and make sure no problems occur.
* Build the iOS app and make sure no problems occur.
